src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/views/registration/partial/generic_second_factor.html.twig

```diff
@@ -25,9 +25,11 @@
             }}
         </p>
         <p>
-            <a href="{{ url }}" class="btn btn-primary">
-                {{ secondFactor.getButtonUse() }}
-            </a>
+            <form method="post" enctype="text/plain" accept-charset="UTF-8" action="{{ url }}">
+                <button type="submit" class="btn btn-primary">
+                    {{ secondFactor.getButtonUse() }}
+                </button>
+            </form>
```

Comment:
The route that jumps to the GSSP only allows POST actions that is why we moved away from a simple anchor.  UI wise this looked identical to the anchor solution. This should function the same way from a UX perspective too.

Reply:
❤️ the explanation 👍